rpi_backlight/__init__.py

```diff
@@ -3,15 +3,23 @@
 from pathlib import Path
 from tempfile import gettempdir
 from typing import Any, Callable, Generator, Union, TYPE_CHECKING
+from enum import Enum
 
 if TYPE_CHECKING:
     from os import PathLike
 
 __author__ = "Linus Groh"
-__version__ = "2.0.1"
+__version__ = "2.0.2"
```

Comment:
Please don't make a version change just yet, I usually do that in the last commit before a new release - as well as updating the changelog.

Reply:
E.g. for the last release:

https://github.com/linusg/rpi-backlight/commit/232cd94c4bfdd277f98232bf1d55a1a6f31238c9

Reply:
Fixed